### PR TITLE
Feature/adding text choose icon

### PR DESCRIPTION
**Pull request by EivinS**

### CODE REVIEW

**Comment by eddiman** [comment]
Remember to add trhe image radio group to all lang files

File: semantics.json [hunk 1]
```diff
@@ -257,6 +257,37 @@
...
+                    "fontFamily": "360-Image",
+                    "options": [
+                      {
+                        "value": "text-icon",
+                        "label": "text",
+                        "fontIcon": "e91f"
+                      },
+                      {
+                        "value": "info-icon",
+                        "label": "info",
```
Comment (eddiman) [inline semantics.json]:
```suggestion
                        "label": "Info icon",
```

File: semantics.json [hunk 2]
```diff
@@ -257,6 +257,37 @@
...
+                        }
+                      ],
+                      "widget": "imageRadioGroup"
+                    },
+                    "alignment": "horizontal",
+                    "fontFamily": "360-Image",
+                    "options": [
+                      {
+                        "value": "text-icon",
+                        "label": "text",
```
Comment (eddiman) [inline semantics.json]:
```suggestion
                        "label": "Text icon",
```

File: scripts/components/Shared/NavigationButton.js [hunk 3]
```diff
@@ -49,7 +50,11 @@ export const getIconFromInteraction = (interaction, scenes) => {
     icon = Icons.AUDIO;
   }
   else if (machineName === 'H5P.AdvancedText') {
-    icon = Icons.INFO_MARK;
+    if(interaction.iconTypeTextBox && interaction.iconTypeTextBox == 'text-icon') {
```
Comment (boyum) [inline scripts/components/Shared/NavigationButton.js]:
```suggestion
    if (interaction.iconTypeTextBox === 'text-icon') {
```

Comment (EivinS) [reply scripts/components/Shared/NavigationButton.js]: sweet

**Review by eddiman — APPROVED** [review]

**Pull request merged**